test/functional/mempool_accept.py

```diff
@@ -333,6 +340,34 @@ def run_test(self):
             maxfeerate=0,
         )
 
+        # Prep for tin-tx tests with wsh(OP_TRUE) output
+        seed_tx = self.wallet.send_to(from_node=node, scriptPubKey=script_to_p2wsh_script(CScript([OP_TRUE])), amount=COIN)
+        self.generate(node, 1)
+
+        self.log.info('A tiny transaction(in non-witness bytes) that is disallowed')
+        tx = CTransaction()
+        tx.vin.append(CTxIn(COutPoint(int(seed_tx[0], 16), seed_tx[1]), b"", SEQUENCE_FINAL))
+        tx.wit.vtxinwit = [CTxInWitness()]
+        tx.wit.vtxinwit[0].scriptWitness.stack = [CScript([OP_TRUE])]
+        tx.vout.append(CTxOut(0, CScript([OP_RETURN] + ([OP_0] * (MIN_PADDING - 2)))))
+        # Note it's only non-witness size that matters!
+        assert_equal(len(tx.serialize_without_witness()), 64)
+        assert_equal(MIN_STANDARD_TX_NONWITNESS_SIZE - 1, 64)
+
+        self.check_mempool_result(
+            result_expected=[{'txid': tx.rehash(), 'allowed': False, 'reject-reason': 'tx-size-small'}],
+            rawtxs=[tx.serialize().hex()],
+            maxfeerate=0,
+        )
+
+        self.log.info('Minimally-small transaction(in on-witness bytes) that is allowed')
```

Comment:
"non"

Reply:
> so this is also testing that additional witness bytes doesn't actually help.

Yes, I only realized this was the case when writing the actual test where I switched to `wsh` to avoid policy jankery. If the test could be more self-documenting, let me know.

Reply:
Um, I just meant that you should have written "non-witness" not "on-witness" ?

Reply:
unsure why I commented *here*, but I was responding to your bigger text!

Reply:
Ah; yes, now I remember realising that when I first read your comment, but apparently it didn't stick

Reply:
fixed